docs/async_service.rst

```diff
@@ -189,7 +194,9 @@ service.  This can be done by passing ``daemon=True`` into the call to
                 ...
 
         async def run(self):
+            # The following two statements are equivalent.
             self.manager.run_task(self.do_long_running_thing, daemon=True)
+            self.manager.run_daemon_task(self.do_long_running_thing)
```

Comment:
Is one of them preferred? Should we discourage use of one as deprecated?

Reply:
Anyone have thoughts on just removing the `daemon=True` option so that you just either use `run_task` or `run_daemon_task` based on your needs?

Reply:
:+1: for removing the boolean `daemon` flag option